backend/backend/views.py

```diff
@@ -30,15 +31,15 @@ class ObtainBearerToken(DRFObtainAuthToken):
     throttle_classes = [AnonRateThrottle, UserLoginThrottle]
 
     def post(self, request, *args, **kwargs):
+        if not settings.EXPIRY_TOKEN_ENABLED:
+            raise GoneException("Implicit login is disabled on this server")
```

Comment:
I understand the logic behind naming this exception, but if it might not make sense to the user unaware of how it is implemented underneath. Why not raise something like `UnsupportedAuthenticationMethodError`?

Reply:
Oh, right, that makes sense for easier debugging.